data/config-schema/src/main/resources/net/corda/schema/configuration/p2p.gateway/1.0/corda.p2p.gateway.json

```diff
@@ -0,0 +1,103 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "$id": "https://corda.r3.com/net/corda/schema/configuration/p2p/1.0/corda.p2p.gateway.json",
+  "title": "Corda P2P Gateway Configuration Schema",
+  "description": "Configuration schema for the P2P gateway.",
+  "type": "object",
+  "properties": {
+    "hostAddress": {
+      "description": "The host name or IP address where the HTTP server will bind.",
+      "type": "string",
+      "format": "hostname",
+      "default": "0.0.0.0"
+    },
+    "hostPort": {
+      "description": "The HTTP port.",
+      "type": "integer",
+      "default": 0,
```

Comment:
I think it would be better to have a constant port as the default, instead of using random assignment. This would also help with our setup (or generally setups), where load balancers might need to know that value in advance. I would use `8080`, which is sort of standard after `80` which might require root privileges.